net/ipv4/tcp_output.c

```diff
@@ -2494,6 +2571,20 @@ static int tcp_mtu_probe(struct sock *sk)
 	}
 	tcp_init_tso_segs(nskb, nskb->len);
 
+#ifdef CONFIG_SECURITY_TEMPESTA
+	if (!skb_tfw_tls_type(nskb) || WARN_ON_ONCE(!sk->sk_write_xmit))
+		goto transmit;
+
+	result = sk->sk_write_xmit(sk, nskb, mss_now, probe_size);
```

Comment:
Basically it's not necessary(seems `mss_now` won't be used in `sk_write_xmit` in case of single skb), but we can replace `mss_now` by `probe_size` just to clarity.